tests/acceptance/blockchain/interfaces/test_testerchain.py

```diff
@@ -96,17 +92,21 @@ def test_testerchain_creation(testerchain, another_testerchain):
 
 
 def test_multiversion_contract():
+
     # Prepare compiler
-    base_dir = os.path.join(dirname(abspath(__file__)), "contracts", "multiversion")
-    v1_dir = os.path.join(base_dir, "v1")
-    v2_dir = os.path.join(base_dir, "v2")
-    root_dir = SolidityCompiler.default_contract_dir()
-    solidity_compiler = SolidityCompiler(source_dirs=[SourceDirs(root_dir, {v2_dir}),
-                                                      SourceDirs(root_dir, {v1_dir})])
+    base_dir = NUCYPHER_TEST_DIR / 'test_contracts' / 'multiversion'
+    v1_dir, v2_dir = base_dir / 'v1', base_dir / 'v2'
+    bundles = (
+        SourceBundle(base_path=v1_dir),
+        SourceBundle(base_path=v2_dir)
+    )
+    compiled_contracts = multiversion_compile(source_bundles=bundles, allow_paths=[base_dir])
 
     # Prepare chain
-    blockchain_interface = BlockchainDeployerInterface(provider_uri='tester://pyevm/2', compiler=solidity_compiler)
-    blockchain_interface.connect()
+    blockchain_interface = BlockchainDeployerInterface(provider_uri='tester://pyevm/2')
+    blockchain_interface.connect(compile_now=False)
+    blockchain_interface._raw_contract_cache = compiled_contracts
```

Comment:
This way to test exclude all multiversion compilation handling inside `BlockchainDeployerInterface` and test becomes less meaningful